build.yml

```diff
@@ -89,8 +101,10 @@ jobs:
           set -ex
           conda build $VERBOSE_OPTION \
             --channel conda-forge ${{ parameters.conda_channels }} \
-            --python=${{ parameters.py_version }} --no-anaconda-upload conda \
-            --override-channels
+            --python=${{ parameters.py_version }} \
```

Comment:
We need to check that packages made via settings for `3.7` can still be installed in envs where python > `3.7`